### PR TITLE
Remove UUID

### PR DESCRIPTION
**Pull request by eatyourgreens**
Remove redundant `for` attributes from form labels, then uninstall `uuid`.

Staging branch URL: https://remove-uuid.pfe-preview.zooniverse.org

Supersedes #4922.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/features/feedback/shared/components/text-input.jsx [hunk 1]
```diff
@@ -1,16 +1,13 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import uuidv4 from 'uuid/v4';
 
 function TextInput({ title, help, onChange, name, type = 'text', required = false, value }) {
-  const uuid = uuidv4();
   return (
     <fieldset>
-      <label htmlFor={uuid}>
+      <label>
```
Comment (srallen) [inline app/features/feedback/shared/components/text-input.jsx]: The input isn't nested in the label here, so wouldn't you need for/id in this case?

File: app/features/feedback/shared/components/select-input.jsx [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
 import React from 'react';
 import Select from 'react-select';
-import uuidv4 from 'uuid/v4';
 
 function SelectInput({ help, onChange, multi = false, name, options, placeholder = '', title, value }) {
-  const uuid = uuidv4();
   return (
     <fieldset>
-      <label htmlFor={uuid}>
+      <label>
```
Comment (srallen) [inline app/features/feedback/shared/components/select-input.jsx]: The select input isn't nested in the label here.

Comment (eatyourgreens) [reply app/features/feedback/shared/components/select-input.jsx]: Good catch. I'll fix that.

**Comment by srallen** [comment]
Is there a threshold that can be set for coveralls? Package updates are even causing failures as you can see in the babel-eslint update PR from dependabot.

**Comment by eatyourgreens** [comment]
I'm not sure. It's always complaining about the same 3 lines in WorkflowSelection too.

**Comment by eatyourgreens** [comment]
By the way, coveralls failing won't block a merge.

**Pull request merged**